parser/src/conds.ts

```diff
@@ -0,0 +1,77 @@
+import * as t from "@babel/types";
+import { Condition } from "xstate";
+import { DeclarationType } from ".";
+import { createParser } from "./createParser";
+import { unionType } from "./unionType";
+import { isFunctionOrArrowFunctionExpression } from "./utils";
+
+export interface CondNode {
+  node: t.Node;
+  name: string;
+  cond: Condition<any, any>;
+  declarationType: DeclarationType;
+  inlineDeclarationId: string;
+}
+
+const CondAsFunctionExpression = createParser({
+  babelMatcher: isFunctionOrArrowFunctionExpression,
+  parseNode: (node, context): CondNode => {
+    return {
+      node,
+      name: "",
+      cond: () => {
+        return false;
+      },
```

Comment:
both of those feel a little bit redundant here

Reply:
See comment above on the weird way I've been testing this